lightning/src/ln/functional_tests.rs

```diff
@@ -2175,15 +2167,25 @@ fn claim_htlc_outputs_single_tx() {
 		}
 
 		let node_txn = nodes[1].tx_broadcaster.txn_broadcasted.lock().unwrap();
-		assert_eq!(node_txn.len(), 29); // ChannelManager : 2, ChannelMontitor: 8 (1 standard revoked output, 2 revocation htlc tx, 1 local commitment tx + 1 htlc timeout tx) * 2 (block-rescan) + 5 * (1 local commitment tx + 1 htlc timeout tx)
-
-		assert_eq!(node_txn[0], node_txn[7]);
-		assert_eq!(node_txn[1], node_txn[8]);
-		assert_eq!(node_txn[2], node_txn[9]);
-		assert_eq!(node_txn[3], node_txn[10]);
-		assert_eq!(node_txn[4], node_txn[11]);
-		assert_eq!(node_txn[3], node_txn[5]); //local commitment tx + htlc timeout tx broadcasted by ChannelManger
+		assert_eq!(node_txn.len(), 26); // ChannelManager : 2, ChannelMontitor: 8 (1 standard revoked output, 2 revocation htlc tx, 1 local commitment tx + 1 htlc timeout tx) * 2 (block-rescan) + 5 * (1 local commitment tx + 1 htlc timeout tx)
+
+		// 3 = 5 = 7 = 9 = 14 = 17 = 20 = 23
```

Comment:
Whats the intuition between these all being the same (can you add a comment for it?)? The comment doesn't seem to indicate they should be.

Reply:
It's for local commitment tx + HTLC-Timeout tx duplicata, right now they are still duplicated (even as of #466, planning to do it in its own PR because transaction generation of this class of txn isn't actually a thing in as-of-master `bump_claim_tx`